### PR TITLE
Handle pre in .note width changes

### PR DESCRIPTION
**Pull request by tomayac**
This PR properly changes the width of code blocks in notes sections.

### CODE REVIEW

File: src/_css/main.css [hunk 1]
```diff
@@ -301,6 +301,9 @@ main a:hover code, main a:focus code {
   color: var(--white);
   background-color: var(--darkblue);
 }
+.note pre[class^="language"] {
```
Comment (mathiasbynens) [inline src/_css/main.css]: Do we need the attribute selector? Wouldn't `.note pre` work?

Comment (tomayac) [reply src/_css/main.css]: It's just a safety net in case someone includes a non-syntax-highlighted `pre` block. It could probably be removed.

Comment (mathiasbynens) [reply src/_css/main.css]: That's what I meant: we'd want those non-syntax highlighted `pre` blocks to also render correctly, right?

Comment (tomayac) [reply src/_css/main.css]: Let me fix it.

**Pull request merged**